rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/attachment/Sw360AttachmentService.java

```diff
@@ -569,4 +575,80 @@ public void fillCheckedAttachmentData(Attachment attachment, User user) throws S
             attachment.unsetCheckedOn();
         }
     }
+
+    public boolean isValidSbomFile(MultipartFile file) throws TException {
+
+        if (file == null || file.isEmpty()) {
+            return false;
+        }
+
+        String originalFilename = file.getOriginalFilename();
+        File tmpFile = null;
+
+        try {
+            String lowerCaseFilename = originalFilename.toLowerCase();
+
+            // SPDX formats
+            if (lowerCaseFilename.endsWith(".spdx") || lowerCaseFilename.endsWith(".rdf")) {
```

Comment:
Please use constants `SW360Constants.RDF_FILE_EXTENSION`, `SW360Constants.XML_FILE_EXTENSION` and create one for `spdx`.